backport

```diff
@@ -135,18 +134,24 @@ if [ "$LOCAL" = "yes" ]; then
   exit 0
 fi
 
-#log_info "Pushing $BRANCH_NAME to origin"
-#git push origin
+if [ "$CI" = "true" ]; then
```

Comment:
After some consideration I would recommend to do not use CI env variable as it's limited to CI/CD environments only. Could we introduce an explicity switch (e.g.`--non-interactive`) instead?

Reply:
> After some consideration I would recommend to do not use CI env variable as it's limited to CI/CD environments only. Could we introduce an explicity switch (e.g.`--non-interactive`) instead?

e20e6ff8cee09b1e4f93143eb3b1a742bc547cc4